pkg/configurator/methods.go

```diff
@@ -161,18 +161,24 @@ func (c *Client) GetOutboundPortExclusionList() []string {
 }
 
 // IsPrivilegedInitContainer returns whether init containers should be privileged
-func (c *Client) IsPrivilegedInitContainer() bool {
-	return c.getConfigMap().EnablePrivilegedInitContainer
+func (c *CRDClient) IsPrivilegedInitContainer() bool {
+	return c.getMeshConfig().EnablePrivilegedInitContainer
 }
 
 // GetConfigResyncInterval returns the duration for resync interval.
 // If error or non-parsable value, returns 0 duration
-func (c *Client) GetConfigResyncInterval() time.Duration {
-	resyncDuration := c.getConfigMap().ConfigResyncInterval
+func (c *CRDClient) GetConfigResyncInterval() time.Duration {
+	resyncDuration := c.getMeshConfig().ConfigResyncInterval
 	duration, err := time.ParseDuration(resyncDuration)
 	if err != nil {
 		log.Debug().Err(err).Msgf("Error parsing config resync interval: %s", duration)
 		return time.Duration(0)
 	}
 	return duration
 }
+
+// GetPermissiveTrafficPolicyMode returns the duration for resync interval.
+// If error or non-parsable value, returns 0 duration
```

Comment:
the comments above this method are incorrect , please update it

Reply:
I didn't notice there is a `IsPermissiveTrafficPolicyMode()`, which serves the same purpose.

This method will be removed.